components/automate-chef-io/content/docs/node-integrations.md

```diff
@@ -199,9 +199,9 @@ Filter the regions for the scan job by specifying regions to include or exclude.
 
 ![Chef Automate Create Azure-API Scan Job](/images/docs/create-azure-api-scanjob.png)
 
-## Google Cloud Platform Account Scanning with Chef Automate (BETA - as of 20181020020209)
+## Google Cloud Platform Account Scanning with Chef Automate
 
-Run scans against your GCP account infrastructure using Chef Automate. Set up Chef Automate to detect and scan the nodes in your Google Cloud Platform (GCP) account by providing your GCP Credentials and creating a _GCP VM Node Manager_. To access the GCP Node Manager, select anywhere on the Automate screen and type 'beta'. After toggling the Google Cloud Node Manager to the "ON" position, close the helper window and then refresh your browser. Now, when you open _Node Integrations_, you should see _Google Cloud_ as one of your node management service options. See the Chef InSpec documentation for more infomation about [resources](https://www.inspec.io/docs/reference/resources/#gcp-resources).
+Run scans against your GCP account infrastructure using Chef Automate. Set up Chef Automate to detect and scan the nodes in your Google Cloud Platform (GCP) account by providing your GCP Credentials and creating a _GCP VM Node Manager_. To create a GCP Node Manager, navigate to _Node Integrations_, click on `Create Integration`, and you should see _Google Cloud_ as one of your node management service options. See the Chef InSpec documentation for more infomation about [resources](https://www.inspec.io/docs/reference/resources/#gcp-resources).
```

Comment:
There's one instance of "GCP Node Manager" in here when it seems like "GCP VM Node Manager" is the terminology we're using.

Reply:
we actually shouldn't be using VM with GCP -- i'll remove the one that says VM

Reply:
(b/c it's only for api scans and vm implies nodes/virtual machines)

Reply:
Any chance we could make that resources link text more descriptive, maybe _GCP Resources_ instead?

Oh also, we tried to remove the word _click_ from the docs as it is centered on people with pointing devices. I believe the docs team decided to use the word _select_ instead.